### DISCUSSION TITLE
Subscribed but still cant use v0.dev

### DISCUSSION THREAD

**Discussion by LymicYT** [vercel/community#4988] [Account Issue]
### Question

I have subscribed, but I'm still being told I'm on the waiting list. Why is this happening?

**Comment by amyegan** [comment]
Hi @LymicYT. Can you walk me through the steps you took to subscribe? And did you already have an invite, or were you still on the waitlist when you subscribed? This sounds like a bug but I want to be sure I investigate the right thing. 🙇

**Reply by LymicYT** [reply]
Hi Amy. I went to https://v0.dev/pricing , and paid there. I was already on the waiting list at the time was not invited. Thanks for replying!

**Comment by LymicYT** [comment]
Hi Amy. I went to https://v0.dev/pricing , and paid there. I was already on the waiting list at the time was not invited. Thanks for replying!
 

Sent: Monday, November 27, 2023 at 3:34 PM

**Reply by amyegan** [reply]
Thanks for sharing this. I reached out to the v0 team for help investigating what went wrong. We'll reach out with more info soon!

**Reply by amyegan** [reply]
The team identified the bug and pushed a solution so no one else ends up in this situation. In your case, they've gone ahead and sent you an invite so you can actually use your subscription. Please let me know if that works for you 🙂

**Comment by LymicYT** [comment]
I’m in! Infact I’ve already had a few goes with it :) thank you for your help!Sent using the mobile mail appOn 27/11/2023 at 23:26, Amy wrote:
                
            
            
                
                    From: "Amy" ***@***.***>Date: 27 November 2023To: "vercel/community" ***@***.***>Cc: "LymicYT" ***@***.***>,"Mention" ***@***.***>Subject: Re: [vercel/community] Subscribed but still cant use v0.dev (Discussion #4988)
                
                
The team identified the bug and pushed a solution so no one else ends up in this situation. In your case, they've gone ahead and sent you an invite so you can actually use your subscription. Please let me know if that works for you 🙂

**Comment by beamtasm** [comment]
This same thing has happened to me.  Subscribed on the pricing page and don't have access.  I've been on the waitlist for a long time.

**Reply by leerob** [reply]
What is your Vercel username?

**Reply by thefarmer2000** [reply]
polarb3ar

(edit: sorry i'm same person as above wrong account, that is my username and I can send you invoice & receipt if you'd like as well.)
